packages/@vuepress/markdown/lib/highlight.js

```diff
@@ -24,12 +24,30 @@ function getLangCodeFromExtension (extension) {
     yml: 'yaml',
     styl: 'stylus',
     kt: 'kotlin',
-    rs: 'rust'
+    rs: 'rust',
+    cs: 'csharp',
+    fs: 'fsharp'
   }
 
   return extensionMap[extension] || extension
 }
 
+function getDocLangCodeFromLang (lang) {
+  const docMap = {
+    'js': 'jsdoc',
+    'javascript': 'jsdoc',
+    'typescript': 'jsdoc',
+    'markup': 'jsdoc',
+    'java': 'javadoc',
+    'php': 'phpdoc',
+    'csharp': 'xml-doc',
+    'fsharp': 'xml-doc',
+    'vbnet': 'xml-doc'
+  }
+
+  return docMap[lang]
+}
+
```

Comment:
Where do these come from? mind providing some context?

Reply:
This was from https://github.com/PrismJS/prism/blob/master/components.json , searching for the keyword `doc` and arranged myself.
The original `PrismJS` PR: https://github.com/PrismJS/prism/pull/1541 https://github.com/PrismJS/prism/pull/2340

PS. I was trying to add `"js":"javascript"` into `getLangCodeFromExtension` just like `"ts":"typescript"` to avoid duplicate of line 37 & 38 but causing unexpected behavior, so I give up